src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -27,36 +28,39 @@ protected override TestWorkspace CreateWorkspace(string fileContents)
             return new CSharpCompletionService(workspace, exclusiveProviders);
         }
 
-        protected override Task BaseVerifyWorkerAsync(
+        private protected override Task BaseVerifyWorkerAsync(
```

Comment:
>BaseVerifyWorkerAsync [](start = 40, length = 21)

Maybe not for this PR: this test classes structure looks complicated. Adding two parameters require changing tons of files. Definitely there should be a context object to handle them all.

Reply:
Yes, this came up in @genlu's review as well, and we decided that we'd handle this in a follow-up PR. Discussion here: https://github.com/dotnet/roslyn/pull/34808/files#r274272141